src/plugins/expressions/public/react_expression_renderer/use_expression_renderer.ts

```diff
@@ -40,6 +44,11 @@ interface ExpressionRendererState {
   error: null | ExpressionRenderError;
 }
 
+const isExpressionError = (result: unknown): result is ExpressionValueError => {
```

Comment:
There is already a helper function called `isExpressionValueError` in the `common` package. Could you please reuse that one?